app/assets/stylesheets/frontend/print/_html-publication.scss

```diff
@@ -0,0 +1,21 @@
+.html-publications-show {
+  nav.in-page-navigation {
+    display: block!important;
```

Comment:
Gosh [that selector](https://github.com/alphagov/static/blob/master/app/assets/stylesheets/helpers/_core-print.scss#L224-L243) seems a little much. Trying to fix that seems outside the scope of this change.